### DISCUSSION TITLE
Clarification on meep-loc-serv api documentation

### DISCUSSION THREAD

**Discussion by lkh5kor** [InterDigitalInc/AdvantEDGE#174] [Q&A] [answered]
Hello,

As per the documentation in swagger.yaml file - https://github.com/InterDigitalInc/AdvantEDGE/blob/master/go-apps/meep-loc-serv/api/swagger.yaml , the base URL is 'https://localhost/location/v2'. However when I try to test the API's in postman using this as the base URL, for example https://localhost/location/v2/queries/users I get 404 page not found as the response. After debugging on it for sometime along with my colleague, we found out that the base URL is should have the sandbox name after localhost. i.e base URL - "https://localhost/<sandbox_name>location/v2/". Can you please make the changes in the documentation as it will be helpful for others who would be using the API's. Kindly let me know the address parameter that needs to be passed to get distance of UE - /queries/distance. The document says the address parameter can be "tel" meaning telephone number. However when I provide my telephone number as the address parameter, I get 501 service not supported as response. I tried using "sip" - (the format of mail id) providing my mail id as address parameter and got the same 501 service not supported response. Kindly let me know what should be the address parameter to be sent. Please refer to the attachments for details. 

![meep-loc-serv-base_url_page_not_found](https://user-images.githubusercontent.com/57914213/103521617-593b4d00-4e79-11eb-8753-871d9196b428.png)
![distance-query-tel-mobile](https://user-images.githubusercontent.com/57914213/103523625-70c80500-4e7c-11eb-804b-fd4ffb30f51e.png)
![distance-query-sip-email-Id](https://user-images.githubusercontent.com/57914213/103523643-77567c80-4e7c-11eb-9497-f49d5e20dbf4.png)

Regards,
Shankar

**Comment by roymx** [comment, marked as answer]
Hello @lkh5kor 

Thanks for pointing out the documentation inconsistency with regards to including sandbox name in the path; the development team will be back from holiday break next week and we will address this.

As for the distance endpoint for MEC013 Location API, it is not yet supported; this should also be indicated clearly in the documentation (which is not at this time).

In the meantime, the list below indicates which location services endpoints are supported and which are not.
![image](https://user-images.githubusercontent.com/48520190/103691864-e8715f00-4f64-11eb-8b33-32c452738d73.png)

Please let us know if some of these endpoints are more critical for your project as we may be able to prioritize support & integration.

**Reply by lkh5kor** [reply]
Hi @roymx ,

Thank you for the detailed response.
I would like to know if the below two API's are implemented and available in AdvantEDGE version 1.6

https://github.com/InterDigitalInc/AdvantEDGE/blob/master/docs/api-metric-engine-notif/Apis/NotificationsApi.md
postNetworkNotification.

Regarding critical endpoints, I will have a discussion with my colleagues and provide you the list of critical endpoints.

Regards,
Shankar

**Reply by dilallkx** [reply]
Hi @lkh5kor ,

Yes, the Metrics Engine subscription & notification APIs are implemented in AdvantEDGE v1.6. You can subscribe for periodic event & network metric notifications by POSTing a new subscription with a notification URL that points to your notification server.

NOTE: The current Metrics Engine implementation uses the notifyURL (provided in the subscription) as a basepath for notifications; you should therefore pass your notification server URL in the subscription and implement the required notification endpoint paths according to the notification API documentation.
